### PR TITLE
non root user and fix book/html calibre

### PR DESCRIPTION
**Pull request by Frooodle**
# Description

non root user and fix book/html calibre

Moves calibre install to ENV

Closes #(issue_number)

## Checklist:

- [x ] I have read the [Contribution Guidelines](https://github.com/Stirling-Tools/Stirling-PDF/blob/main/CONTRIBUTING.md)
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] My changes generate no new warnings

## Contributor License Agreement

By submitting this pull request, I acknowledge and agree that my contributions will be included in Stirling-PDF and that they can be relicensed in the future under the MPL 2.0 (Mozilla Public License Version 2.0) license.

(This does not change the general open-source nature of Stirling-PDF, simply moving from one license to another license)

### CODE REVIEW

**Comment by sbplat** [comment]
In the future, we should try using something like [this](https://semantic-release.gitbook.io/semantic-release/) to automatically handle releases.

File: src/main/java/stirling/software/SPDF/model/ApplicationProperties.java [hunk 1]
```diff
@@ -289,24 +289,7 @@ public String toString() {
...
-            }
-
-            @Override
-            public String toString() {
-                return "CustomApplications [installBookAndHtmlFormats="
-                        + installBookAndHtmlFormats
-                        + "]";
-            }
-        }
+        public static class CustomApplications {}
```
Comment (sbplat) [inline src/main/java/stirling/software/SPDF/model/ApplicationProperties.java]: Is this still needed?

Comment (Frooodle) [reply src/main/java/stirling/software/SPDF/model/ApplicationProperties.java]: Not needed but kept just if we want to add in future, but yes its un-used

Comment (Frooodle) [reply src/main/java/stirling/software/SPDF/model/ApplicationProperties.java]: let me remove actually

Comment (Frooodle) [reply src/main/java/stirling/software/SPDF/model/ApplicationProperties.java]: removed

**Review by sbplat — APPROVED** [review]

**Pull request merged**